hydromt/drivers/preprocessing.py

```diff
@@ -0,0 +1,76 @@
+"""Common routines for preprocessing."""
+from typing import Callable, Dict
+
+import numpy as np
+import xarray as xr
+
+
+def round_latlon(ds: xr.Dataset, decimals: int = 5) -> xr.Dataset:
```

Comment:
No tests were added for this file.

Reply:
not retesting copied-over functionality

Reply:
Why not? Copying means testing to my opinion, or copy the tests.

Reply:
I renamed the file, not copying functionality.